src/utils/formsg-utils.ts

```diff
@@ -7,13 +7,35 @@ export function getField(
 ): string | undefined {
   const response = responses.find(({ question }) => question === name)
 
-  return response?.answer
+  return response?.answer?.trim()
+}
+
+function trimAllStrings(
+  responseArray: string[] | string[][]
+): string[] | string[][] {
+  if (Array.isArray(responseArray)) {
+    responseArray.map((item) => {
+      if (Array.isArray(item)) {
+        return trimAllStrings(item)
+      }
+      if (typeof item === "string") {
+        return item.trim()
+      }
+      return item
+    })
+    return responseArray
+  }
+  return responseArray
```

Comment:
Thoughts on reducing the checks here, given that the method signature is for string[] | string[][]?

Reply:
TS complains if we dont have this because it is not able to infer that 
if (Array.isArray(responseArray)), then responseArray is of type []. I have added in a comment for clarity here: 
[08433bb](https://github.com/isomerpages/isomercms-backend/pull/698/commits/08433bb83269ceb74044f3e9e7a6dc6e1c4fd4e8)